spec/lib/reports/report_metadata_builder_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'active_support/testing/time_helpers'
+
+RSpec.describe Reporting::ReportMetadataBuilder do
+  include ActiveSupport::Testing::TimeHelpers
```

Comment:
We have the time helpers active for all specs now. No need to include this.
```suggestion

RSpec.describe Reporting::ReportMetadataBuilder do
```